packages/cloud2edge/Chart.yaml

```diff
@@ -25,9 +25,16 @@ keywords:
 home: https://eclipse.org/packages/packages/cloud2edge/
 icon: https://eclipse.org/packages/packages/cloud2edge/images/c2e-logo.svg
 sources:
-  - https://github.com/eclipse/packages/packages/cloud2edge/
+  - https://github.com/eclipse/packages/tree/master/packages/cloud2edge
```

Comment:
FMPOV this should point to the standard repository URL

Reply:
To me the link to the actual chart source directory looks more convenient. Going to the parent repository from there is trivial.
Starting out with the repository URL however, it's not directly obvious in which directory the cloud2edge chart is defined.
(Kafka and other Bitnami charts also link to the chart sub-directory in the sources link, btw.)

Reply:
Ok, now I see your point. I was under the impression that the original link actually pointed to the correct place but it seems to result in a 404 ...